wiki/Mascots/en.md

```diff
@@ -1,234 +1,99 @@
-A video about the mascots: <https://www.youtube.com/watch?v=mJF2cAs_MrI>
+# Mascots
 
-osu! Mascots
-============
+*See also: [Mascots/Gallery](/wiki/Mascots/Gallery).*
 
-Pippi
------
+A YouTube video showcasing the osu! mascots can be seen at [Mascot Showcase](https://youtu.be/mJF2cAs_MrI).
 
-![Pippi](Pippi_Daru_Header.png "Pippi")
+## Official
 
-#### Real Name
-Pippi
+### pippi
 
-#### Alisaes
-* Pippi
-* Pippidon
+![pippi](img/pippi.png "pippi")
 
-#### Birthday
-?
+pippi, stylized with a lowercase p, is the osu!standard mascot that joined on 2008-07. She is also known as pippidon in osu!taiko and had an appeareance in [Yandere Simulator](https://yanderesimulator.com) as an NPC. The initial concept art was created by Sarumaru, the pippidon sprite were created by [crystalsuicune](/users/9974), and the current art was designed by [Daru](/users/32480).
```

Comment:
with a lowercase `P`
had an appearance -> appeared
sprite was/sprites were
link to Sarumaru's profile?

Reply:
> with a lowercase `P`

going with `with a lowercase "p"`

> link to Sarumaru's profile?

I'm not sure what his user id is.

Reply:
isnt it incorrect with ASG then?